content/doc/tutorials/index.adoc

```diff
@@ -68,6 +68,7 @@ You may find more tutorials in link:/node/tags/tutorial[Tutorial blogposts].
 * link:/blog/2017/02/07/declarative-maven-project/[Declarative Pipeline for Maven Projects]
 * link:/blog/2016/08/10/rails-cd-with-pipeline/[Continuous Security for Rails apps with Pipeline and Brakeman]
 * link:/blog/2016/08/29/sauce-pipeline/[Browser-testing with Sauce OnDemand and Pipeline]
+* link:using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline[Build a Jenkins pipeline by using Jenkinsfile Runner GitHub Actions]
```

Comment:
```suggestion
* link:using-jenkinsfile-runner-github-action-to-build-jenkins-pipeline[Build a Jenkins pipeline by using Jenkinsfile Runner GitHub Actions]
```
This section is specifically for **tutorial blog posts**, so this wouldn't be added there unless it is submitted as a blog post. If the goal is to have a tutorial blog post, it would need to be formatted according to the [contributing guide](https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post)

Reply:
I see. I think this one is the tutorial for a tool so it should be moved to the **tools** section.

Reply:
Resloved this problem in the latest commits.